tests/cypress/page-object/ckeditor5.ts

```diff
@@ -59,7 +59,7 @@ export class RichTextCKeditor5Field extends Field {
             .should('be.visible')
             .find('button.ck-button')
             .contains(label)
-            .should('be.visible')
+            .should('exist')
```

Comment:
Buttons might not always display labels, so just check for existence to be able to fetch by label